README.md

```diff
@@ -6,6 +6,23 @@ has been developed using Python 3.6. This is a work in progress.
 The repo contains a bash shell script that can be used to install the dependencies, as well as build the application and
 run the tests. Below are the installation instructions if not using the bash script.
 
+## Local config module
+
+You will need to create an `instance/config.py` file. For local development,
+it should contain the following:
+
+```python
+DEBUG = True
+
+# The root location of static assets (Javascript, CSS, Images)
+STATIC_ROOT = 'http://localhost:9000'
+
+# If this is not set, live reload will be disabled.
+LIVE_RELOAD_PATH = 'ws://localhost:9000/ws'
```

Comment:
How is this used? I'm not finding it in standard Flask and I don't see any new python dependencies.